common/pom.xml

```diff
@@ -104,6 +104,41 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
```

Comment:
As I said elsewhere, we should copy the exceptions, so we shouldn't depend on ZK here.